src/pymovements/dataset/dataset_files.py

```diff
@@ -536,9 +537,20 @@ def load_precomputed_event_file(
     if custom_read_kwargs is None:
         custom_read_kwargs = {}
 
-    valid_extensions = {'.csv', '.tsv', '.txt'}
-    if data_path.suffix in valid_extensions:
+    csv_extensions = {'.csv', '.tsv', '.txt'}
+    r_extensions = {'.rda'}
+    valid_extensions = csv_extensions | r_extensions
+    if data_path.suffix in csv_extensions:
         precomputed_event_df = pl.read_csv(data_path, **custom_read_kwargs)
+    elif data_path.suffix in r_extensions:
+        if 'r_dataframe_key' in custom_read_kwargs:
+            precomputed_r = pyreadr.read_r(data_path)
```

Comment:
It would be much better to have a `from_rdata()` function that is well documented than this hidden in here.

However, this could also be improved on later when we get rid of all the `precomputed_` stuff, so it doesn't matter much as the function will be removed anyway.